server/metrics.go

```diff
@@ -73,5 +75,14 @@ func NewServerMetrics(cfg Config) *Metrics {
 			Name:      "inflight_requests",
 			Help:      "Current number of inflight requests.",
 		}, []string{"method", "route"}),
+		SlowRequestServerThroughput: reg.NewHistogramVec(prometheus.HistogramOpts{
+			Namespace:                       cfg.MetricsNamespace,
+			Name:                            "slow_request_server_throughput_" + cfg.ThroughputConfig.Unit,
+			Help:                            "Server throughput of long running requests.",
+			ConstLabels:                     prometheus.Labels{"cutoff_ms": strconv.FormatInt(cfg.ThroughputConfig.SlowRequestCutoff.Milliseconds(), 10)},
+			NativeHistogramBucketFactor:     cfg.MetricsNativeHistogramFactor,
```

Comment:
let's add the default prometheus buckets too. I think without them we wouldn't export classic histograms. Because this is a library it would make sense to still support the old format